### PR TITLE
Fix the script to take domain arguments

### PR DESCRIPTION
**Pull request by dharada1**
Thank you @josemlwdf 👍

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ To run the script, execute it from the command line:
 
 Example
 
-    python3 extract_streams.py sample.pcap "microsofto365.com"
+    python3 dnscat_decoder.py sample.pcap "microsofto365.com"
```
Comment (dharada1) [inline README.md:35]: script name is different

File: dnscat_decoder.py [hunk 2]
```diff
@@ -32,15 +32,15 @@ def check_tshark_installed():
             print("Installation aborted.")
 
 
-def extract_tcp_streams_from_pcap(pcap_file):
+def extract_tcp_streams_from_pcap(pcap_file, bad_domain):
```
Comment (dharada1) [inline dnscat_decoder.py:35]: to take any domain other than `microsofto365com`

File: README.md (hunk 1 above)
Comment (josemlwdf) [reply README.md:35]: Thanks for spotting that

**Review by josemlwdf — APPROVED** [review]
Set up domain as a variable, not just a fix domain fot HTB Sherlock :)

**Pull request merged**